framework/auth/__init__.py

```diff
@@ -60,6 +63,13 @@ def register_unconfirmed(username, password, fullname):
             password=password,
             fullname=fullname)
         user.save()
+        mails.queue_mail(
```

Comment:
Since the `NO_ADDON` email is being added to the queue when a user registers for the OSF, it's not going to be sent to existing users that do not have any addons configured. Is that expected?